### ISSUE TITLE
FlxBar recycle problem

### ISSUE THREAD

**Issue by nshen** [photonstorm/Flixel-Power-Tools#5]
@photonstorm hi ,because of the constructor parameters, FlxBar cant be recycled by the FlxGroup,can the parameters move to a init method ?

**Comment by photonstorm** [comment]
Good point - I will see about moving them, or at least giving them default values and then adding an init method.
